### DISCUSSION TITLE
missing permission to invoke 'StorageAttach' when executing lotus worker run

### DISCUSSION THREAD

**Discussion by Sakuya-CN** [filecoin-project/lotus#6866] [General Q&A] [answered]
I have a permission problem when executing lotus worker run <flag>

The information is as follows:
2021-07-25T03:09:07.062Z	INFO	main	lotus-seal-worker/main.go:336	Opening local storage; connecting to master
2021-07-25T03:09:07.063Z	WARN	main	lotus-seal-worker/main.go:98	opening path /mnt/Params/lotus-worker/lotusworker:
    github.com/filecoin-project/lotus/extern/sector-storage/stores.(*Local).open
        /mnt/Params/lotus/extern/sector-storage/stores/local.go:233
  - declaring storage in index:
    github.com/filecoin-project/lotus/extern/sector-storage/stores.(*Local).OpenPath
        /mnt/Params/lotus/extern/sector-storage/stores/local.go:212
  - missing permission to invoke 'StorageAttach' (need 'admin')

I don't know how to solve this problem. Can someone help me? thank you

**Comment by Sakuya-CN** [comment, marked as answer]
有人能帮助我吗

**Reply by de-authority** [reply]
Same problem here: https://filecoinproject.slack.com/archives/CEGN061C5/p1627409499034800
I think this is a permission problem. Do the users running lotus worker have permission to use the storage of this application
